tests/ui/invalid-compile-flags/crate-type-flag.rs

```diff
@@ -30,7 +30,7 @@
 //@[bin] check-pass
 
 //@[proc_dash_macro] ignore-wasm (proc-macro is not supported)
-//@[proc_dash_macro] compile-flags: --crate-type=proc-macro
+//@[proc_dash_macro] compile-flags: --crate-type=proc-macro -C panic=unwind
```

Comment:
Mind adding `//@[proc_dash_macro] needs-unwind` to ignore this test when panic=unwind is not supported by the target or codegen backend?

Reply:
Yeah, please just add //@ needs-unwind to check if that is sufficient, the -C panic=unwind is entirely irrelevant to this test and is a distraction. Alternatively, just ignore-fuchsia.